mathesar_ui/src/component-library/help/Help.scss

```diff
@@ -3,7 +3,7 @@ button.dropdown.trigger.no-arrow.help-trigger {
 }
 
 .help-trigger-content {
-  color: #39a0f5;
+  color: var(--brand-500);
```

Comment:
Also, this change turns help bubbles red, making them look like errors or warnings. Blue is a color that seems to be commonly associated with help messages. Can we keep it blue (and find a suitable on-brand shade of blue)?

Reply:
Help bubbles are red in the Figma design, this should be an action item for @ghislaineguerin.

Reply:
I'd prefer that we use any of the gray (slate) shadings instead of blue. In most examples I've found of other apps it's always a color close to that of the body text.

Reply:
Looks good. Thanks.